listings/ch00-introduction/if_else/src/access_control.cairo

```diff
@@ -1,23 +1,30 @@
-use starknet::ContractAddress;
-
 #[contract]
 mod SimpleAccessControl {
-    use starknet::get_caller_address;
+    use starknet::{get_caller_address,ContractAddress};
 
     struct Storage {
-        owner: starknet::ContractAddress, 
+        owner: ContractAddress, 
     }
 
+    #[event]
+    fn welcomeEvent(name: felt252) {}
+
     #[constructor]
-    fn constructor(_address: starknet::ContractAddress) {
+    fn constructor(_address: ContractAddress) {
         owner::write(_address);
     }
 
+    #[internal]
```

Comment:
In that case, the `#[internal]` keyword is not required - everything that is neither `#[external]` nor `#[view]` is internal by default.